src/main/java/com/microsoft/sqlserver/jdbc/DDC.java

```diff
@@ -893,16 +901,9 @@ static final Object convertTemporalToObject(JDBCType jdbcType, SSType ssType, Ca
             default:
                 throw new AssertionError("Unexpected SSType: " + ssType);
         }
```

Comment:
why do this instead of our usual SQLServerException.makeFromDriverError and use an error string?

Reply:
It's just the way code was before, which I didn't touch. I think we can leave this as is.